packages/es-components/src/components/controls/buttons/DropdownButton.js

```diff
@@ -135,6 +135,7 @@ function DropdownButton(props) {
   const [buttonValue, setButtonValue] = useState(props.buttonValue);
   const [isOpen, setIsOpen] = useState(false);
 
+  const initialRender = useRef(true);
```

Comment:
One of these days I would like to review with you why this works @Darrken

Reply:
Normally you'd use `useRef` to store a reference to a DOM element, but it can be used to store *any* mutable value. In this case I'm using it to just store the boolean. I chose this over using state to avoid re-renders on the change of value.